src/main/java/com/lineate/mdzinarishvili/reminderapp/ReminderAppApplication.java

```diff
@@ -12,24 +12,31 @@
 @SpringBootApplication
 public class ReminderAppApplication {
 
-    public static void main(String[] args) {
-        SpringApplication.run(ReminderAppApplication.class, args);
-    }
+  public static void main(String[] args) {
+    SpringApplication.run(ReminderAppApplication.class, args);
+  }
 
-    @Bean
-    public CommandLineRunner commandLineRunner(
-            AuthenticationService service
-    ) {
-        return args -> {
-            var admin = RegisterRequest.builder()
-                    .username("Admin")
-                    .email("admin@mail.com")
-                    .password("password")
-                    .role(ADMIN)
-                    .build();
-            System.out.println("Admin token: " + service.register(admin).getAccessToken());
+  @Bean
+  public CommandLineRunner commandLineRunner(
+      AuthenticationService service
+  ) {
+    return args -> {
+      var admin = RegisterRequest.builder()
+          .username("Admin")
+          .email("admin@mail.com")
+          .password("password")
+          .role(ADMIN)
+          .build();
+      var other = RegisterRequest.builder()
+          .username("3938484")
+          .email("a234234dmin@mail.com")
+          .password("password")
+          .role(ADMIN)
+          .build();
```

Comment:
Use logger
Add @Slf4j under the class name and then you can use log.info("message");